x/gov/simulation/operations.go

```diff
@@ -273,7 +279,11 @@ func operationSimulateMsgVote(ak types.AccountKeeper, k keeper.Keeper,
 		msg := types.NewMsgVote(simAccount.Address, proposalID, option)
 
 		account := ak.GetAccount(ctx, simAccount.Address)
-		fees, err := simulation.RandomFees(r, ctx, account.SpendableCoins(ctx.BlockTime()))
+		balances := bk.GetAllBalances(ctx, account.GetAddress())
+		locked := bk.LockedCoins(ctx, account.GetAddress())
+		spendable := balances.Sub(locked)
```

Comment:
Think there should be a `bk.GetSpendable(acc)` function to avoid this boilerplate everywhere

Reply:
Indeed. I think it's fair to add that API back into the bank keeper. Note, I have `operations.go` modified in the unit tests PR (https://github.com/cosmos/cosmos-sdk/pull/5580). I'll make those changes there.

Reply:
Done: https://github.com/cosmos/cosmos-sdk/pull/5580/commits/ac3df34e857e270f2ce9087798e65426b478972c